tgstation.dme

```diff
@@ -1,6 +1,7 @@
 // DM Environment file for tgstation.dme.
 // All manual changes should be made outside the BEGIN_ and END_ blocks.
 // New source code should be placed in .dm files: choose File/New --> Code File.
+// interface\skin.dmf needs to be at the very end of the file.
```

Comment:
why? byond appears to be trying to force it to the top, we should not have people editing in dreammaker cause breakage to the dme. 

please work out a way to work around this if it is what I suspect, rather then enforcing it with a line of text

Reply:
Okay.

Reply:
thanks. I don't wanna be too aggressive here, this is a real cool change, I just really don't like deprecating dreammaker, which seems? like is what's going on

Reply:
didnt see this either, not good =(